### PR TITLE
ILM indices names

### PR DESCRIPTION
**Pull request by dome313**
it was not clear how the indices created by ILM going to look like, so I added the description

### CODE REVIEW

File: libbeat/docs/shared-ilm.asciidoc [hunk 1]
```diff
@@ -61,6 +61,7 @@ overwrite the template to apply the changes.
 ==== `setup.ilm.pattern`
 
 The rollover index pattern. The default is `%{now/d}-000001`.
+Indices created with this pattern have name `%{[setup.ilm.rollover_alias]}-%{now/d}-000001`
```
Comment (dedemorton) [inline libbeat/docs/shared-ilm.asciidoc]:
```suggestion
Indices created with this pattern have the name `%{[setup.ilm.rollover_alias]}-%{now/d}-000001`
```

**Comment by dome313** [comment]
:+1

**Comment by dome313** [comment]
👍

**Comment by dome313** [comment]
👍

**Comment by bmorelli25** [comment]
Oh wow, sorry this was missed for so long, @dome313. Unfortunately, `7.4` is no longer an actively maintained version of our documentation. I tried to change the base of this PR to `master`, but it doesn't look like that's going to worth. If you want to open a new PR and point the change at `master`, I can get it merged for you. If you'd rather I do that, I'm happy to as well. Just let me know! Thanks, and sorry again.

**Pull request closed without merging**